bin/node/cli/src/service.rs

```diff
@@ -26,18 +26,19 @@ use node_primitives::Block;
 use node_runtime::RuntimeApi;
 use sc_client_api::{ExecutorProvider, RemoteBackend};
 use sc_consensus_babe::{self, SlotProportion};
+use sc_executor::NativeExecutor;
 use sc_network::{Event, NetworkService};
 use sc_service::{config::Configuration, error::Error as ServiceError, RpcHandlers, TaskManager};
 use sc_telemetry::{Telemetry, TelemetryWorker};
 use sp_runtime::traits::Block as BlockT;
 use std::sync::Arc;
 
-type FullClient = sc_service::TFullClient<Block, RuntimeApi, Executor>;
+type FullClient = sc_service::TFullClient<Block, RuntimeApi, NativeExecutor<Executor>>;
```

Comment:
Are we tying ourselves more to a Native Executor here? (Presumably there will be clients that want to just use a wasm executor?)

Reply:
We're trying the node into the native executor, but other clients can just set the type definition differently